mantidproject/mantid
# Fix failing doctests

**Description of work.**

Updated doctests to take into account other code changes. 

**To test:**
1. Build workbench and `AllTests` target. Then open workbench and in ManangeUserDirectories add the paths to the UnitTests and DocTests in the test data. Close workbench.
2. Build the `docs-doctest` target and check all tests pass

Fixes #35073 


**Note: One of the jenkins pipelines will need updating to add the `--enable-doctests` to the buildscript call. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
